src/getFunctionName.lua

```diff
@@ -0,0 +1,16 @@
+local function getFunctionName(fn)
+	local functionName
+	if typeof(fn) == "function" then
+		-- try using Roblox Lua's debug.info before falling back to lua5.1 debug.getinfo
+		-- selene: allow(incorrect_standard_library_use)
+		functionName = (type(debug.info) == "function" and debug.info(fn, "sln"))
+			or (type(debug.getinfo) == "function" and debug.getinfo(fn).source)
+	end
+	if functionName == nil or functionName == "" then
+		functionName = tostring(fn)
```

Comment:
I can't find anything in the docs about whether `tostring()` can error-out, but this caught my attention.  If there is no failure case, ignore this comment :)

Reply:
not for a function, no. the worse case here is it gives `function: 0xdeadb33f`